### PR TITLE
Hapi v17

### PR DESCRIPTION
**Pull request by JordanHood**
#### What does this PR do? (please provide any background)

https://hxshortbreaks.atlassian.net/browse/NB-699

this updates this plugin to work on hapi v17
#### How can this be tested?

see https://github.com/holidayextras/the-works/pull/705
#### What gif best describes how you feel about this work?

- I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

By approving a review you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---

### CODE REVIEW

File: lib/pluginElephant.js [hunk 1]
```diff
@@ -62,22 +62,27 @@ function pluginElephant (server, options, next) {
...
     console.log(error)
-    next()
   }
 }
 
 exports.register = pluginElephant
-exports.register.attributes = {
-  pkg: require('../package')
+const pkg = require('../package.json')
+const name = pkg['name']
```
Comment (t1gr0u) [inline lib/pluginElephant.js]: `pkg.name`?

File: lib/pluginElephant.js [hunk 2]
```diff
@@ -62,22 +62,27 @@ function pluginElephant (server, options, next) {
...
+const version = pkg['version']
```
Comment (t1gr0u) [inline lib/pluginElephant.js]: `pkg.version`?

File: package.json [hunk 3]
```diff
@@ -26,21 +26,25 @@
   },
   "dependencies": {
     "elephant": "git+ssh://git@github.com:holidayextras/elephant.git#v3.10.1",
-    "lodash": "4.15.0",
-    "q": "1.0.1"
+    "is-my-json-valid": "^2.17.1",
```
Comment (t1gr0u) [inline package.json]: no caret in `dependencies`

Comment (JordanHood) [reply package.json]: i dont know where that came from as that package itsn't used, ill remove it

**Pull request merged**